### ISSUE TITLE
There are no precompiled binaries for node 12

### ISSUE THREAD

**Issue by kondi** [kesla/node-snappy#172]
Please upload precompiled binaries for node 12 on windows x64.
Based on this comment we should have: https://github.com/kesla/node-snappy/issues/167#issuecomment-486210167

```
$ npm install snappy

> snappy@6.2.2 install D:\projects\test\node_modules\snappy
> prebuild-install || node-gyp rebuild

prebuild-install WARN install No prebuilt binaries found (target=12.4.0 runtime=node arch=x64 libc= platform=win32)
```

**Comment by kesla** [comment]
Thanks for the bug report, I'm trying out a solution in #173!

**Comment by kesla** [comment]
@kondi fixed as of `v6.2.3`

**Comment by kondi** [comment]
Thank you for the quick update!
